packages/gestalt/src/SearchField.css

```diff
@@ -17,7 +17,7 @@
 }
 
 .input::placeholder {
-  color: #8e8e8e;
+  color: #6c6c6c;
```

Comment:
Is this something we should fo for `<TextArea />` and `<TextField />` as well? The placeholder there is also the old gray
```
.textArea::placeholder {
  color: #8e8e8e;
}
```

Reply:
Nope. Keeping the old placeholder on those because they have a white background. It's slightly inconsistent, but accessibility is the north star on this decision.